templates/assets/js/main.js

```diff
@@ -0,0 +1,82 @@
+const right_edge = '<div class="col-md-1 d-flex" id="right_edge" style="padding: 0;margin: 0;padding-top: 15px;padding-bottom: 15px;opacity: 1;">' +
+		'<div class="w-100 rounded-right" style="background-color: rgba(0,0,0,0.7);padding-top: 0;padding-bottom: 0;"></div>' +
+		'</div>';
+
+function ajaxCall(url, data, method, successCallback, errorCallback) {
```

Comment:
Why did you wrap the ajax call ?

Reply:
For my opinion It's more legible to manage the JS Ajax call in one function other than copy the same code (like the dataType) all over the application.

Reply:
Wrapping the ajax call limits its functionallity.
If you would want to send an additional info with the ajax call, you will need to change the function signature and fix its call everywhere you used it.
This problem actually taking effect here because you should add contentType to the list of parameters ajax is taking.
The default contentType is "application/x-www-form-urlencoding", and we will need to send "application/json" as well.

Reply:
@DoRTaL94 same here

Reply:
I'm waiting for @eidoil32 response.